pldoc/_components/buttons.md

```diff
@@ -102,10 +90,10 @@ info:          Buttons should be used for performing actions within the edX envi
     <a href="http://ux.edx.org/" class="btn">View the UX Pattern Library</a>
     <a href="http://ux.edx.org/" class="btn-neutral">View the UX Pattern Library</a>
     <a href="http://ux.edx.org/" class="btn-brand">View the UX Pattern Library</a>
-    <a href="http://ux.edx.org/" class="btn-elevated">View the UX Pattern Library</a>
 </div>
 
 <h3 class="example-set-hd">Links without visual styling</h3>
+<p>To indicate 'read' or 'visited' links in content areas, use color Primary Dark.</p>
```

Comment:
I find myself confused by this. I read this as advice to the user of the Pattern Library, but this isn't something in their control. Maybe you can make it sound less like an option, such as

    Note: unread links use the primary color, while read/visited links use primary dark.